cluster/gce/upgrade.sh

```diff
@@ -102,6 +100,10 @@ function wait-for-master() {
 function prepare-upgrade() {
   ensure-temp-dir
   detect-project
+  # TODO(mbforbes): Do we still need get-password?
```

Comment:
I don't think we need the password for anything. The nodes don't ever get the password, and the master should have it on the data disk even if the boot vm is replaced.